packages/components/api/cached-apis/data/index.js

```diff
@@ -0,0 +1,25 @@
+export { schemas } from './schemas.js'
```

Comment:
api should be in search component until there is a need to elevate it.  we should only have one package that is 'search'.  The package is a component but not need to have /components in packages.

Reply:
cached-apis is already external, remove this, add the npm packaged @action-agenda/cached-apis.

Reply:
Done!